### DISCUSSION TITLE
video background color selection

### DISCUSSION THREAD

**Discussion by leprincep35700** [senshu/Sozi#593] [Ideas]
the video generated as a black backgroud outside.
A good think should be a background color selection (minimum black or white)
best regards
Patrick Leprince

**Comment by aumouvantsillage** [comment]
I usually recommend to set the background color in Inkscape.

1. Open the *Document Properties* dialog from the *File* menu.
2. At the bottom of the *Page* tab, choose the desired *Background color*.

The default background in Inkscape is fully transparent. As a consequence, the actual background color when displaying the SVG document depends on the viewing software.
Web browsers use a white background for web pages, while the video exporter defaults to black.
